pkg/dss/cockroach/store.go

```
@@ -3,10 +3,68 @@ package cockroach
 import (
 	"context"
 	"database/sql"
+	"database/sql/driver"
+	"time"
 
+	"github.com/golang/protobuf/ptypes"
 	_ "github.com/lib/pq" // Pull in the postgres database driver
+	dspb "github.com/steeling/InterUSS-Platform/pkg/dssproto"
+	"go.uber.org/multierr"
 )
 
+type nullTime struct {
+	Time  time.Time
+	Valid bool
+}
+
+func (nt *nullTime) Scan(value interface{}) error {
+	nt.Time, nt.Valid = value.(time.Time)
+	return nil
```

do we want to return an error here at scan time? Also would time.Parse be useful?

---

Fair point, I'll adjust to handle the case of `value` being `nil` separately, and error out if the type assertion fails for another reason.

---

Done!